### DISCUSSION TITLE
ASAP7 Synthesis using OpenROAD Yosys.

### DISCUSSION THREAD

**Discussion by nerdylye** [ucb-bar/chipyard#1566] [Q&A]
Hi, I am a new user here. I realized the ASAP7 tutorial in the documentation require commercial tools. Is there a way to run ASAP7 with OpenROAD? Thanks in advance

**Comment by harrisonliew** [comment]
Unfortunately, not at this moment. We developed the tutorials before the OpenROAD team worked on ASAP7 support. This is quite low on our priority list, because we focus more on Sky130 now, but we'd appreciate any community contributions to Hammer and the tutorial in Chipyard.

**Reply by nerdylye** [reply]
I see, thank you for letting me know 😄

**Comment by exhaust-create** [comment]
Hello, have you successfully combined ASAP7 with openroad? If you made it, could you tell me how to achieve this?
